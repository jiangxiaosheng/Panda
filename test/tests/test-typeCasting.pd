func main() {
	var x = float(3)
}