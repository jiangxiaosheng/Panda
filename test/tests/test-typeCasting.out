

Sementically checked program: 


void main() {
var x: float(float : 3)
}

